deployer/template/aws_elasticloadbalancingv2_targetgroup.go

```diff
@@ -0,0 +1,35 @@
+package template
+
+import (
+	"github.com/awslabs/goformation/cloudformation"
+	"github.com/awslabs/goformation/cloudformation/resources"
+)
+
+func ValidateAWSElasticLoadBalancingV2TargetGroup(
+	projectName, configName, resourceName string,
+	template *cloudformation.Template,
+	res *resources.AWSElasticLoadBalancingV2TargetGroup,
+) error {
+	res.Name = normalizeName("fenrir", projectName, configName, resourceName, 32)
+
+	res.Tags = append(res.Tags, resources.Tag{Key: "ProjectName", Value: projectName})
+	res.Tags = append(res.Tags, resources.Tag{Key: "ConfigName", Value: configName})
+	res.Tags = append(res.Tags, resources.Tag{Key: "ServiceName", Value: resourceName})
+
+	// Only allow lambda targets for now
```

Comment:
Why only lambda targets? I think we should allow instance or lambda, so that we can test deploying lambdas and asgs behind an ALB in development

Reply:
this was just to make it easier for an MVP

Reply:
now includes instance targets